d3a_api_client/redis_device.py

```diff
@@ -54,7 +56,7 @@ def _aggregator_response_callback(self, message):
     def _check_transaction_id_cached_out(self, transaction_id):
         return transaction_id in self._transaction_id_buffer
 
-    def select_aggregator(self, aggregator_uuid, is_blocking=True):
+    def select_aggregator(self, aggregator_uuid, is_blocking=True, should_unsubscribe=True):
```

Comment:
Minor, but IMO this flag naming is misleading. I would propose: 'unsubscribe_from_device_events'.